### ISSUE TITLE
request: tooling to validate layer specs

### ISSUE THREAD

**Issue by andresgnlez** [Vizzuality/layer-manager#128]
I think it would be quite useful that LM offers a set of tools to validate, in an isolated way, an incoming layer spec and export warning/errors that might happen during the validation process.

This tool could be used both in LM (and avoid render wrong layers leading to tricky errors coming from Mapbox and deeper dependencies) and for specific workflows in applications (like layer creation/edition in a back-office)

**Comment by Bluesmile82** [comment]
I started a layer type checker for Mapbox layers. Maybe it can help as a start https://github.com/Vizzuality/jupyter-layer-test

**Comment by andresgnlez** [comment]
@Bluesmile82 yup, I think it's a good idea! Perhaps we should start defining the scope, features and boundaries of this tool to ensure everyone is on the same page.
